docs/getting-started/installing-service-binding.md

```diff
@@ -8,7 +8,17 @@ sidebar_position: 1
 
 ## Operator
 
-_How can To be provided soon_
```

Comment:
Hey @DhritiShikhar , on this section, we should have instructions that are covering both OpenShift and Kubernetes distributions.

Reply:
Okay

Reply:
I have added information about installation on both Kubernetes and Openshift